experiments/ClimaEarth/setup_run.jl

```diff
@@ -134,6 +124,9 @@ function CoupledSimulation(config_dict::AbstractDict)
         parameter_files,
     ) = get_coupler_args(config_dict)
 
+    # Initialize communication context (do this first so all printing is only on root)
+    comms_ctx = Utilities.get_comms_context(config_dict)
```

Comment:
I'll move this up to be the first thing done in this function, so that any printing in the arg parsing is only on root